pkgs/top-level/all-packages.nix

```diff
@@ -4315,8 +4315,6 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
-  patdiff = callPackage ../tools/misc/patdiff { };
```

Comment:
Could we keep this as a top-level alias like we do for other CLI tools that happen to be written in OCaml?

Removing it completely is a breaking change for folks who are using it:

```
error: undefined variable 'patdiff'
```

Here's a PR: https://github.com/NixOS/nixpkgs/pull/185003/